qt/scientific_interfaces/Inelastic/BayesFitting/CMakeLists.txt

```diff
@@ -1,8 +1,24 @@
 get_filename_component(SUB_DIRECTORY "${CMAKE_CURRENT_LIST_DIR}" NAME)
 
-set(SRC_FILES BayesFitting.cpp BayesFittingTab.cpp Quasi.cpp ResNorm.cpp Stretch.cpp)
+set(SRC_FILES
+    BayesFitting.cpp
+    BayesFittingTab.cpp
+    Quasi.cpp
+    ResNormModel.cpp
+    ResNormView.cpp
+    ResNormPresenter.cpp
+    Stretch.cpp
+)
 
-set(MOC_FILES BayesFitting.h BayesFittingTab.h Quasi.h ResNorm.h Stretch.h)
+set(MOC_FILES
+    BayesFitting.h
+    BayesFittingTab.h
+    Quasi.h
+    ResNormModel.h
+    ResNormPresenter.h
```

Comment:
Can the Model and Presenter go in the `INC_FILES` section because they should contain no Qt?

Reply:
Maybe its not possible to move the Presenter yet because it still needs to inherits from the BayesFittingTab which is a QObject?

Reply:
I was going to do, I forgot at some point. Thanks.